test/csuite/timestamp_abort/main.c

```diff
@@ -395,7 +396,8 @@ thread_run(void *arg)
                 if (i % PREPARE_YIELD == 0)
                     __wt_yield();
                 testutil_check(__wt_snprintf(tscfg, sizeof(tscfg),
-                  "commit_timestamp=%" PRIx64 ",durable_timestamp=%" PRIx64, active_ts, active_ts));
+                  "commit_timestamp=%" PRIx64 ",durable_timestamp=%" PRIx64, active_ts,
+                  i % PREPARE_DURABLE_AHEAD_COMMIT == 0 ? active_ts + 2 : active_ts));
```

Comment:
This will make half of the prepared transactions' durable timestamp larger than their commit timestamp and prepared timestamp.